detekt/detekt.bzl

```diff
@@ -13,36 +13,36 @@ def _impl(ctx):
         "-Xms16m",
         "-Xmx128m",
         "-cp",
-        ":".join([jar for jar in [ctx.file._detekt_cli_jar.path, ctx.file._detekt_wrapper_jar.path]]),
+        ":".join([jar.path for jar in [ctx.file._detekt_cli_jar, ctx.file._detekt_wrapper_jar]]),
         "io.buildfoundation.bazel.rulesdetekt.wrapper.Main",
     ]
 
-    if ctx.attr.config != None:
-        action_inputs.append(ctx.file.config)
+    if ctx.attr.config is not None:
```

Comment:
Identity is generally more preferred than equality with singleton objects since it is a bit faster. PEP 8 has [a mention](https://www.python.org/dev/peps/pep-0008/#programming-recommendations) but not a lot of motivation.

Reply:
LOL, Starlark does not support it, reverting.